src/app/portlets/content-types/fields/service/field.service.ts

```diff
@@ -29,40 +29,35 @@ export class FieldService {
      * @memberof FieldService
      */
     saveFields(contentTypeId: string, fields: Field[]): Observable<any> {
-        const observables: Observable<any>[] = fields.map((field, index) => {
-            const fieldToSend = Object.assign({}, field, {
-                contentTypeId: contentTypeId,
-                sortOrder: index + 1
-            });
+        fields.forEach((field, index) => {
+            field.contentTypeId = contentTypeId;
 
-            if (FieldUtil.isColumn(fieldToSend) || FieldUtil.isRow(fieldToSend)) {
-                fieldToSend.name = `fields-${index}`;
+            if (FieldUtil.isColumn(field) || FieldUtil.isRow(field)) {
```

Comment:
Here we can use the method I mention here: https://github.com/dotCMS/core-web/pull/412/files#diff-9e80d14b75239bd0ca5269f23a9406e7R22